src/interfaces/Helpers.ts

```diff
@@ -1 +1,6 @@
-export type mapToProps = (state: any, ownProps?: any) => any;
+import Store from "./Store";
+
+export type mapToProps = (state: object, ownProps?: object) => object;
+export type ActionsBase<State> = (
```

Comment:
Why is this being exported? I think it's not being used

Reply:
there are utilities you can use in your definitions
I need to add some examples to use it.
if you want, I can remove it for now and later(with examples) re-add it

Reply:
Yes, please do that.

Reply:
done